packages/connectivity_plus_web/lib/src/utils/connectivity_result.dart

```diff
@@ -5,17 +5,14 @@ import 'package:connectivity_plus_platform_interface/connectivity_plus_platform_
 ConnectivityResult networkInformationToConnectivityResult(
   html.NetworkInformation info,
 ) {
-  if (info == null) {
-    return ConnectivityResult.none;
```

Comment:
not quite sure about this

Reply:
@miquelbeltran is the null-safety expert here, but I _think_ this is fine :smile: